Code/Editor/CMakeLists.txt

```diff
@@ -163,6 +163,8 @@ ly_add_target(
         editor_files.cmake
     PLATFORM_INCLUDE_FILES
         Platform/${PAL_PLATFORM_NAME}/editor_${PAL_PLATFORM_NAME_LOWERCASE}.cmake
+    TARGET_PROPERTIES
+        LY_INSTALL_GENERATE_RUN_TARGET TRUE
```

Comment:
We should add this for the AssetProcessor as well.
Probably it would also be good to add it to the AssetBuilder, AssetBundler, Atom MaterialEditor and Atom ShaderManagmentConsole applications for good measure.

Reply:
I also forgot, probably should add it to the ProjectManager as well

Reply:
Maybe @lawsonamzn has an opinion there.

Not many users run the AP or any of those other exes. All those tools should be available from the Editor.

Reply:
We can easily add that later by just adding the property to it.  Lets start out with just the editor and see if there's any demand for more.  

**or would adding the asset processor fix "builder only" dependencies perhaps?**

Reply:
The AssetProcessor having a custom_target wouldn't fix the builder only dependency issue.
The generation of the gem load dependencies is based on the existance of the target in CMake, not any downstream build solutions.
So because `AssetProcessor` target exist, it still creates a `cmake_dependencies.<project>.assetprocessor.setreg` file in the project directory.